src/app/common/StatusBadge.vue

```diff
@@ -50,7 +41,10 @@ const props = defineProps({
   },
 })
 
-const statusObject = computed(() => STATUS[props.status])
+const statusObject = computed(() => ({
+  title: i18n.t(`http.api.value.${props.status}`),
+  appearance: STATUS[props.status],
+}))
```

Comment:
Could we just use the http property as the classname i.e. `not_available` vs `not-available` (i.e. use `props.status`) then we can get rid of this extra code (and the above `STATUS`)? In fact this would almost be a pure javascript-less presentational component then

Reply:
Fine by me. I’ve changed this now.